packages/framework/src/Component/Image/ImageFactory.php

```diff
@@ -17,6 +41,36 @@ public function create(
         ?string $type,
         ?string $temporaryFilename
     ): Image {
-        return new Image($entityName, $entityId, $type, $temporaryFilename);
+        $temporaryFilePath = $this->fileUpload->getTemporaryFilepath($temporaryFilename);
```

Comment:
?string $temporaryFilename $temporaryFilePath = $this->fileUpload->getTemporaryFilepath($temporaryFilename);
can we discuss this change a bit ?

Reply:
The commit message is wrong, it should have been "from Service to Factory" (not Facade) https://github.com/shopsys/shopsys/commit/830f51d7bf07e04b2430b5d92775d8eccb9ffe36

Click into the commit and find changes in `ImageService`, this logic was moved to `ImageFactory`

Reply:
I will take a look at it... :wink: